core/network/src/consensus_gossip.rs

```diff
@@ -357,7 +334,6 @@ impl<B: BlockT> ConsensusGossip<B> {
 					entry.remove_entry();
 				}
 			}
-			self.multicast_inner(protocol, message_hash, topic, status, || message.clone());
```

Comment:
it doesn't re-multicast in case of new incoming message?

Reply:
no, the validator is supposed to call `broadcast_message` if it needs to be propagated furter.